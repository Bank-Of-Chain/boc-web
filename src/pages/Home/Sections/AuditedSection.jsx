import React from "react"
// @material-ui/core components
import { makeStyles } from "@material-ui/core/styles"

import styles from "./auditedStyle"

const useStyles = makeStyles(styles)

export default function TeamSection () {
  const classes = useStyles()
  return (
    <div className={classes.section}>
      <h2 className={classes.title}>Audited And Verified</h2>
      <div className={classes.container}>
        <h2 className={classes.text}>
          <img className={classes.img} src={require("./../images/un-audit.png")} alt=''></img>This audit is in the
          testing phase...
        </h2>
      </div>
    </div>
  )
}
