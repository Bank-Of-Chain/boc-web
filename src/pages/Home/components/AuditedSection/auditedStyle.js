const auditedStyle = {
  section: {
    color: '#1F2023',
    padding: '16rem 0',
    background: 'url(/images/home/bg-gradient.png) no-repeat center'
  },
  sectionMobile: {
    color: '#1F2023',
    padding: '5rem 2.75rem',
    background: 'url(/images/home/bg-gradient.png) no-repeat center'
  },
  inner: {
    maxWidth: '75rem',
    margin: '0 auto'
  },
  title: {
    margin: 0,
    fontSize: '6.25rem',
    lineHeight: '8.125rem'
  },
  '@media screen and (max-width: 923px)': {
    title: {
      fontSize: '4.6rem',
      lineHeight: '6rem'
    }
  },
  text: {
    width: '34rem',
    margin: 0,
    lineHeight: '1.875rem',
    fontSize: '1.25rem',
    fontWeight: 'normal'
  },
  divider: {
    width: '18.75rem',
    height: '1px',
    margin: '1.875rem 0',
    background: '#1F2023'
  }
}

export default auditedStyle
