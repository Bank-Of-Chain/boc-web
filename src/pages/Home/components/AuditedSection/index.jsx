import React from 'react'
import { makeStyles } from '@material-ui/core/styles'
import useMediaQuery from '@material-ui/core/useMediaQuery'

import styles from './auditedStyle'

const useStyles = makeStyles(styles)

export default function TeamSection() {
  const classes = useStyles()
  const isLayoutSm = useMediaQuery('(max-width: 960px)')
  return (
    <div className={isLayoutSm ? classes.sectionMobile : classes.section}>
      <h1 className={classes.title}>Your assets,</h1>
      <h1 className={classes.title}>secured</h1>
      <div className={classes.divider}></div>
      <h4 className={classes.text}>
        The underlying smart contracts of the BoC protocol have been thoroughly and repeatedly tested. BoC contracts are currently under our first
        third-party audit.
      </h4>
    </div>
  )
}
