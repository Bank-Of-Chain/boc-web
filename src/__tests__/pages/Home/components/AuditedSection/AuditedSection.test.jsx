import React from 'react'
import { render } from '@testing-library/react'
import AuditedSection from '@/pages/Home/components/AuditedSection'
import { describe, expect, it } from 'vitest'

describe('xxxxxxxx', () => {
  it('AuditedSection Component Render', () => {
    const { asFragment } = render(<AuditedSection />)
    expect(asFragment()).toMatchSnapshot()
  })
})
