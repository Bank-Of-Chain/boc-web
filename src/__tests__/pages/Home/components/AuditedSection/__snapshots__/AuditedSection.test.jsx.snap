// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`xxxxxxxx > AuditedSection Component Render 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-section-1"
  >
    <div
      class="makeStyles-inner-3"
    >
      <h1
        class="makeStyles-title-4"
      >
        Your assets,
      </h1>
      <h1
        class="makeStyles-title-4"
      >
        secured
      </h1>
      <div
        class="makeStyles-divider-6"
      />
      <h4
        class="makeStyles-text-5"
      >
        The underlying smart contracts of the BoC protocol have been thoroughly and repeatedly tested. BoC contracts are currently under our first third-party audit.
      </h4>
    </div>
  </div>
</DocumentFragment>
`;
